pom.xml

```diff
@@ -35,11 +35,20 @@
     <java.level>8</java.level>
   </properties>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+      </dependency>
```

Comment:
Without a `version`, what does this even do?

Reply:
Resolve a dependency conflict, it takes the version from the parent so does not need version.

```
[WARNING] Rule 5: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.slf4j:slf4j-api:1.7.26 paths to dependency are:
+-org.jenkins-ci.modules:sshd:2.8-SNAPSHOT
  +-org.apache.sshd:sshd-core:2.5.1
    +-org.apache.sshd:sshd-common:2.5.1
      +-org.slf4j:slf4j-api:1.7.26 (managed) <-- org.slf4j:slf4j-api:1.7.30
and
+-org.jenkins-ci.modules:sshd:2.8-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.249.1
    +-org.slf4j:jcl-over-slf4j:1.7.26
      +-org.slf4j:slf4j-api:1.7.26 (managed) <-- org.slf4j:slf4j-api:1.7.30
and
+-org.jenkins-ci.modules:sshd:2.8-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.249.1
    +-org.slf4j:log4j-over-slf4j:1.7.26
      +-org.slf4j:slf4j-api:1.7.26 (managed) <-- org.slf4j:slf4j-api:1.7.30
and
+-org.jenkins-ci.modules:sshd:2.8-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-war:2.249.1
    +-org.slf4j:slf4j-jdk14:1.7.26
      +-org.slf4j:slf4j-api:1.7.26 (managed) <-- org.slf4j:slf4j-api:1.7.30
]

```

Reply:
Hmm, interesting.

Reply:
this shouldn't be needed after bumping the core version

Reply:
I hope not. So long as this remains a module, you might as well use the latest weekly as `jenkins.version`.